src/views/Login.vue

```diff
@@ -74,10 +76,17 @@ export default defineComponent({
       const instanceURL = this.instanceUrl.trim().toLowerCase();
       if(!this.baseURL) this.store.dispatch("user/setUserInstanceUrl", this.alias[instanceURL] ? this.alias[instanceURL] : instanceURL);
       const { username, password } = this;
-      this.store.dispatch("user/login", { username, password }).then((data: any) => {
+      this.store.dispatch("user/login", { username, password }).then(async (data: any) => {
         if (data.token) {
           this.username = ''
           this.password = ''
+
+          // Get product identification from api using dxp-component and set the state if eComStore is defined
+          if(this.currentEComStore.productStoreId){
```

Comment:
This should be done asynchronous in login action

Reply:
done